README.md

```diff
@@ -97,14 +97,16 @@ make push run
 This is 2 step process. First copy lldb-server binary to mobile and then from the host machine connect to the server
 
 ### copy lldb-server to mobile and make it listen 
+```
 $ adb push ~/Android/Sdk/ndk/<ver>/toolchains/llvm/prebuilt/linux-x86_64/lib/clang/19/lib/linux/aarch64/lldb-server /sdcard
 $ adb shell run-as org.yourorg.cnfgtest cp /sdcard/lldb-server /data/data/org.yourorg.cnfgtest/
 $ adb shell run-as org.yourorg.cnfgtest chmod +x /data/data/org.yourorg.cnfgtest/lldb-server
 $ adb shell 'run-as org.yourorg.cnfgtest /data/data/org.yourorg.cnfgtest/lldb-server platform --listen *:1234 --server
-
+```
 ### connect to lldb-server and debug
-$ ~/Android/Sdk/ndk/<ver>/toolchains/llvm/prebuilt/linux-x86_64/bin/lldb.sh -o 'platform select remote-android' -o 'platform connect  connect://192.168.1.189:1234'
-
+```
+$ ~/Android/Sdk/ndk/<ver>/toolchains/llvm/prebuilt/linux-x86_64/bin/lldb.sh -o 'platform select remote-android' -o 'platform connect  connect://192.168.1.189:1234' -o "attach `adb shell pidof org.yourorg.cnfgtest`"
```

Comment:
```suggestion
$ ~/Android/Sdk/ndk/<ver>/toolchains/llvm/prebuilt/linux-x86_64/bin/lldb.sh \
    -o 'platform select remote-android' \
    -o 'platform connect  connect://192.168.1.189:1234' \
    -o "attach `adb shell pidof org.yourorg.cnfgtest`"
```

Reply:
In general I'd also avoid <ver> as placeholder in bash context because a mindless copy and paste would make that a redirect which could create or in worst case overwrite a file. But in this case I believe that would fail or won't do any serious harm.